scep/scep_test.go

```diff
@@ -130,6 +130,16 @@ func TestSignCSR(t *testing.T) {
 	testParsePKIMessage(t, certRep.Raw)
 }
 
+type testNewCSRRequestStruct struct {
+	keyUsage        x509.KeyUsage
+	shouldCreateCSR bool
+}
+
+var testNewCSRRequestData = []testNewCSRRequestStruct{
```

Comment:
I'd move both the struct and variable inside the function and refactor the for loop to use t.Run. 
Example here (t.Parallel optional): https://github.com/golang/go/wiki/TableDrivenTests#parallel-testing

Reply:
Thanks for letting me know about that. That's way cleaner :-)